pulse_actions/email_config.cfg

```diff
@@ -0,0 +1,6 @@
+{
+    "sender_list": ["vaibhavmagarwal@gmail.com"],
```

Comment:
Right now, I have added my email id only. Let me know if any one else would like to subscribe to the errors.